src/components/Modals/NewCourse/NewCourseModal.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <flexible-modal
     title="Add Course"
-    contentClass="content-course"
+    content-class="content-course"
```

Comment:
Isn't the required prop name `contentClass`?

Reply:
Also some case transformation on props